semgrep-core/finding/unit_files.ml

```diff
@@ -15,7 +15,8 @@ let unittest =
         let filters = Files_filter.mk_filters
           ~excludes:["*.{c,h}"; "*.go"]
           ~includes:["foo.*"]
-          ~exclude_dirs:["c"] in
+          ~exclude_dirs:["c"] 
+          ~include_dirs:["a/b"] in
```

Comment:
This will not work, because include_dirs, like exclude_dirs, apply the filter to the directory component of a path,
it's not to a prefix of a path, so here you want ~include_dirs:["a"] (or ~include_dirs:["b"]).
GREP exclude dirs work like that I think.
I know @DrewDennison wanted to change that, but then you can have issues when the path is ./a/b/c
and if you put "a/b" it will not match because it's not a prefix.